### PR TITLE
MGMT-2328: always require user reboot by default

### PR DESCRIPTION
**Pull request by RazRegev**
Currently the master nodes not ready due failure to configure CNI.
For now, until fixing this issue we will skip the reset state and always
require manual reboot on reset.

### CODE REVIEW

File: subsystem/cluster_test.go [hunk 1]
```diff
@@ -1486,81 +1485,6 @@ var _ = Describe("cluster install", func() {
 					Expect(host.Bootstrap).Should(Equal(false))
 				}
 			})
-			It("[only_k8s]reset cluster and disable bootstrap", func() {
```
Comment (avishayt) [inline subsystem/cluster_test.go:1489]: Assuming we'll fix the reset one day, why not leave the existing unit tests for EnableAutoReset=True, and add new ones for EnableAutoReset=False?

**Review by eranco74 — COMMENTED** [review]
Why not change IsRequireUserActionReset ?
If you like you can add the Config.EnableAutoReset there.

**Review by eranco74 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**